### DISCUSSION TITLE
RGB ledstrip

### DISCUSSION THREAD

**Discussion by snikhaas** [frntc/SIDKick-pico#42] [Q&A]
Hi,

Can i use a ledstrip If I use the RGB led firmware ?

If so , wich pins on the pico  do I need for addressing the led strip
![20240414_193454](https://github.com/frntc/SIDKick-pico/assets/68494412/f574f8bd-437c-4ff8-93c1-1c988304555c)

**Comment by frntc** [comment]
The SKpico can use the RGB LED (WS2812) which is available on some Pico clones (connected to GPIO23). On the original Pico, GPIO23 is used differently, and in both cases it's not available at a pin. You can use such clones and grab the DI data signal from the PCB and connect it to your stripe (if it's a WS2812 or compatible). But the firmware only controls one RGB LED.

**Reply by snikhaas** [reply]
@frntc Thanks for checking in.

Would it be possible with a custom firmware ?  

Would be a nice feature.. 

Have a nice day,
Gr,
Marc
